shipstation_integration/customer.py

```diff
@@ -177,17 +179,20 @@ def create_customer(order: "ShipStationOrder"):
         frappe.log_error(title="Error saving Shipstation Customer", message=e)
 
 
-def create_contact(order: "ShipStationOrder", customer_name: str):
-    contact = frappe.get_value("Contact Email", {"email_id": customer_name}, "parent")
+def create_contact(order: "ShipStationOrder", email_id: str=None, phone_no: int=None):
```

Comment:
Does Shipstation send it as a long-int? Or string? Otherwise it should be:

```suggestion
def create_contact(order: "ShipStationOrder", email_id: str=None, phone_no: str=None):
```